LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_LaserSchematics.uc

```diff
@@ -336,7 +336,8 @@ static function X2DataTemplate CreateTemplate_Bullpup_Laser_Schematic()
 	`CREATE_X2TEMPLATE(class'X2SchematicTemplate', Template, 'Bullpup_LS_Schematic');
 
 	Template.ItemCat = 'weapon';
-	Template.strImage = "img:///UILibrary_LW_LaserPack.Inv_Laser_Bullpup";
```

Comment:
Can you use a place holder here ie laser smg, this asset does not exist. Add a TODO as well